src/app/hat-claim/hat-claim-details/hat-claim-details.component.ts

```diff
@@ -15,10 +15,18 @@ import {Component, Input, OnInit} from '@angular/core';
 })
 export class HatClaimDetailsComponent implements OnInit {
   @Input('email') public email: string;
+  public errorInputMsg = 'Unspecified email address';
 
   constructor() { }
 
   ngOnInit() {
   }
+  public getEmailValueOrError(): string {
```

Comment:
Can simply replace the whole method with a default Angular component input value. Read this SO for reference: https://stackoverflow.com/questions/36071942/how-to-set-default-values-for-angular-2-component-properties